themes/mlibrary_new/items/show.php

```diff
@@ -1,5 +1,63 @@
 <?php
-fire_plugin_hook('public_items_show', array('view' => $this, 'item' => $item));
+ /**
+  * Copyright (c) 2016, Regents of the University of Michigan.
+  * All rights reserved. See LICENSE.txt for details.
+  */
+  $item_title = strip_formatting(metadata('item', array('Dublin Core', 'Title')));
+  if (empty($item_title)) { $item_title = __('[Untitled]'); }
+
+  echo head(
+    array(
+      'title'     => $item_title,
+      'bodyid'    => 'items',
+      'bodyclass' => 'show item'
+    )
+  );
+ // if the item is part of the exhibit builder layout
+ if (isset($_GET['exhibit']) && isset($_GET['page'])) {
+    $pageClean = html_escape($_GET['page']);
+    $exhibitClean = html_escape($_GET['exhibit']);
+    echo '<div class="exhibit-item-back-button"><a href="' .
+        exhibit_builder_exhibit_uri(
+          get_record_by_id('exhibit', $exhibitClean),
```

Comment:
From what I could tell, $exhibit was already populated in the view.

Reply:
The code will be updated to check if you user coming from Exhibit page or Gallery page.